src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/CSharpFormattingPass.cs

```diff
@@ -71,8 +71,6 @@ public async override Task<FormattingResult> ExecuteAsync(FormattingContext cont
             cancellationToken.ThrowIfCancellationRequested();
 
             // We make an optimistic attempt at fixing corner cases.
-            var cleanupChanges = CleanupDocument(changedContext);
-            changedText = changedText.WithChanges(cleanupChanges);
             changedContext = await changedContext.WithTextAsync(changedText);
```

Comment:
This line can go too, and the comment, and one of the `ThrowIfCancellationRequested` calls.